### PR TITLE
Update chain.json

### PR DESCRIPTION
**Pull request by BccNodes**

### CODE REVIEW

File: nois/chain.json [hunk 1]
```diff
@@ -327,6 +339,12 @@
     ]
   },
   "explorers": [
+    {
+      "kind": "BccNodes",
+      "url": "https://explorer.bccnodes.com/nois-M",
```
Comment (JeremyParish69) [inline nois/chain.json:344]: Explorer doesn't seem to show any data?

**Comment by dylanschultzie** [comment]
@BccNodes please provide an appropriate PR title

**Review by dylanschultzie — CHANGES_REQUESTED** [review]
Please update PR title

**Comment by JeremyParish69** [comment]
inactive PR

**Pull request closed without merging**